core/packages/test/scripts/helpers/generateHexEncodedCallData.js

```diff
@@ -0,0 +1,146 @@
+const fs = require("fs");
+const { exit } = require("process");
+const { WsProvider, ApiPromise } = require("@polkadot/api");
+const util = require("@polkadot/util");
+
+// connect to a substrate chain and return the api object
+async function connect(endpoint, types = {}) {
+  const provider = new WsProvider(endpoint);
+  const api = await ApiPromise.create({
+    provider,
+    types,
+    throwOnConnect: false,
+  });
+  return api;
+}
+
+function writeHexEncodedBytesToOutput(method, outputFile) {
+  console.log("Payload (hex): ", method.toHex());
+  console.log("Payload (bytes): ", Array.from(method.toU8a()));
+  fs.writeFileSync(outputFile, JSON.stringify(Array.from(method.toU8a())));
+}
+
+function remarkWithEvent(endpoint, outputFile) {
+  console.log(`Generating remarkWithEvent from RPC endpoint: ${endpoint} to outputFile: ${outputFile}`);
+  connect(endpoint)
+    .then((api) => {
+      const call = api.tx.system.remarkWithEvent("Hello");
+      writeHexEncodedBytesToOutput(call.method, outputFile);
+      exit(0);
+    })
+    .catch((e) => {
+      console.error(e);
+      exit(1);
+    });
+}
+
+function addExporterConfig(endpoint, outputFile, bridgedNetwork, bridgeConfig) {
+  console.log(`Generating addExporterConfig from RPC endpoint: ${endpoint} to outputFile: ${outputFile} based on bridgedNetwork: ${bridgedNetwork}, bridgeConfig: ${bridgeConfig}`);
+  connect(endpoint)
+    .then((api) => {
+      const call = api.tx.bridgeTransfer.addExporterConfig(JSON.parse(bridgedNetwork), JSON.parse(bridgeConfig));
```

Comment:
Taken from cumulus repo but edited here to treat `bridgedNetwork` as JSON as it can be a complex type such as `{ "Ethereum": { "chainId": 1 } }` as opposed to a simple string like "Wococo".